### PR TITLE
bump v1.4.0-beta.2 to v1.4.0-beta.3

### PR DESCRIPTION
**Pull request by xhebox**
Signed-off-by: xhe <xw897002528@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/releases/release-1.4.0-beta.3.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## 新功能
+
+- TiProxy的实验性支持 ([#4693](https://github.com/pingcap/tidb-operator/pull/4693)), [@xhebox](https://github.com/xhebox)
+
+## 错误修复
+
+- 修复错误信息中的拼写错误 ([#4773](https://github.com/pingcap/tidb-operator/pull/4773), [@dveeden](https://github.com/dveeden))
+
+- 修复清理卷快照备份失败问题 ([#4783](https://github.com/pingcap/tidb-operator/pull/4783), [@fengou1](https://github.com/fengou1))
```
Comment (ran-huang) [inline zh/releases/release-1.4.0-beta.3.md:19]:
```suggestion
- 修复清理卷快照备份失败的问题 ([#4783](https://github.com/pingcap/tidb-operator/pull/4783), [@fengou1](https://github.com/fengou1))
```

File: zh/releases/release-1.4.0-beta.3.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+- 修复大规模TiKV节点 (40+) 下备份tidb集群失败 ([#4784](https://github.com/pingcap/tidb-operator/pull/4784), [@fengou1](https://github.com/fengou1))
```
Comment (ran-huang) [inline zh/releases/release-1.4.0-beta.3.md:21]:
```suggestion
- 修复大规模 TiKV 节点 (40+) 下备份 TiDB 集群失败的问题 ([#4784](https://github.com/pingcap/tidb-operator/pull/4784), [@fengou1](https://github.com/fengou1))
```

File: en/releases/release-1.4.0-beta.3.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+# TiDB Operator 1.4.0-beta.3 Release Notes
+
+Release date: December 2, 2022
+
+TiDB Operator version: 1.4.0-beta.3
+
+## New Feature
+
+- Experimental support for TiProxy ([#4693](https://github.com/pingcap/tidb-operator/pull/4693)), [@xhebox](https://github.com/xhebox)
```
Comment (ran-huang) [inline en/releases/release-1.4.0-beta.3.md:13]:
```suggestion
- Add experimental support for TiProxy ([#4693](https://github.com/pingcap/tidb-operator/pull/4693)), [@xhebox](https://github.com/xhebox)
```

File: en/releases/release-1.4.0-beta.3.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## Bug fixes
+
+- Fix typo in error messages ([#4773](https://github.com/pingcap/tidb-operator/pull/4773), [@dveeden](https://github.com/dveeden))
+
+- Fix volume-snapshot backup cleanup failure ([#4783](https://github.com/pingcap/tidb-operator/pull/4783), [@fengou1](https://github.com/fengou1))
```
Comment (ran-huang) [inline en/releases/release-1.4.0-beta.3.md:19]:
```suggestion
- Fix the issue of volume-snapshot backup cleanup failure ([#4783](https://github.com/pingcap/tidb-operator/pull/4783), [@fengou1](https://github.com/fengou1))
```

File: en/releases/release-1.4.0-beta.3.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+- Fix backup failure on tidb cluster has massive tikv nodes (40+) ([#4784](https://github.com/pingcap/tidb-operator/pull/4784), [@fengou1](https://github.com/fengou1))
```
Comment (ran-huang) [inline en/releases/release-1.4.0-beta.3.md:21]:
```suggestion
- Fix backup failure when the TiDB cluster has massive TiKV nodes (40+) ([#4784](https://github.com/pingcap/tidb-operator/pull/4784), [@fengou1](https://github.com/fengou1))
```

**Review by KanShiori — APPROVED** [review]

File: zh/releases/release-1.4.0-beta.3.md [hunk 6]
```diff
@@ -12,6 +12,11 @@ TiDB Operator 版本：1.4.0-beta.3
 
 - 实验性支持 TiProxy ([#4693](https://github.com/pingcap/tidb-operator/pull/4693)), [@xhebox](https://github.com/xhebox)
 
+- 基于 Amazon EBS 的 TiDB 集群 volume-snapshot 备份和恢复 GA ([#4784](https://github.com/pingcap/tidb-operator/pull/4784), [@fengou1](https://github.com/fengou1))，此功能有以下特点：
+
+    - 将备份对 QPS 的影响降至小于 5%
+    - 缩短备份和恢复时间
```
Comment (fengou1) [inline zh/releases/release-1.4.0-beta.3.md:18]: - 缩短备份和恢复时间 （如：备份一小时，恢复两小时）
    
主要是提供信息什么是缩短。

**Review by fengou1 — APPROVED** [review]
Rest LGTM

File: en/releases/release-1.4.0-beta.3.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+TiDB Operator version: 1.4.0-beta.3
+
+## New Feature
+
+- Add experimental support for TiProxy ([#4693](https://github.com/pingcap/tidb-operator/pull/4693)), [@xhebox](https://github.com/xhebox)
+
+- Snapshot backup and restore based on Amazon EBS becomes GA ([#4784](https://github.com/pingcap/tidb-operator/pull/4784), [@fengou1](https://github.com/fengou1)). This feature has the following benefits:
+
+    - Reduce the impact of backup on QPS to less than 5%
+    - Shorten the backup and restore time
```
Comment (shichun-0415) [inline en/releases/release-1.4.0-beta.3.md:18]:
```suggestion
    - Back up and restore data in a short time. For example, finish backup within 1 hour and restore in 2 hours.
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**